modules/ROOT/pages/secure-configuration-properties.adoc

```diff
@@ -272,37 +272,58 @@ image::secure-configuration-properties-studio.png[config extension]
 [[secure_props_tool]]
 == Secure Properties Tool
 
-You can download the https://docs.mulesoft.com/downloads/mule-runtime/4.1/secure-properties-tool.jar[JAR file] for this tool so you can encrypt or decrypt single values, and complete files (both, YAML and Properties files). You can run it in the command line as follows:
+You can download the https://docs.mulesoft.com/downloads/mule-runtime/4.1/secure-properties-tool.jar[JAR file] for this tool so you can encrypt or decrypt single values, and complete files (both, YAML and Properties files).
+This tool supports three encryption modes: `string`, `file`, and `file-level`
+
+=== Encrypt or Decrypt Single values
+
+Use the `string` mode to encrypt or decrypt a single value provided as a parameter.
```

Comment:
Use the `string` mode to encrypt or decrypt a single value provided as a parameter, *for example:* 
You can also just add a : at the end of the sentence.